velox/functions/lib/window/tests/WindowTestBase.cpp

```diff
@@ -48,6 +49,79 @@ WindowTestBase::QueryInfo WindowTestBase::buildWindowQuery(
   return {op, functionSql, querySql};
 }
 
+namespace {
+std::vector<std::string> commaSplit(const std::string& overClause) {
+  std::istringstream tokenStream(overClause);
+  std::string token;
+  std::vector<std::string> splits;
+  while (std::getline(tokenStream, token, ',')) {
+    splits.push_back(token);
+  }
+
+  return splits;
+}
+}; // namespace
+
+WindowTestBase::QueryInfo WindowTestBase::buildStreamingWindowQuery(
+    const std::vector<RowVectorPtr>& input,
+    const std::string& function,
+    const std::string& overClause,
+    const std::string& frameClause) {
+  std::string functionSql =
+      fmt::format("{} over ({} {})", function, overClause, frameClause);
+  core::PlanNodePtr op = nullptr;
+  std::vector<std::string> orderByClauses;
+
+  auto partitionByStartIdx = overClause.find(kPartitionByStr);
```

Comment:
Nit : Rename variables to partitionByIndex and orderByIndex.

Reply:
Updated.